base/linalg/generic.jl

```diff
@@ -795,6 +795,19 @@ function inv(A::AbstractMatrix{T}) where T
     A_ldiv_B!(factorize(convert(AbstractMatrix{S}, A)), eye(S0, checksquare(A)))
 end
 
+function pinv(v::AbstractVector{T}, tol::Real=real(zero(T))) where T
```

Comment:
Should probably be `tol::Real=eps(real(float(one(T))))*length(v)` for consistency with `pinv(::StridedMatrix{T})` (and likewise below)